compiler/il/ILOpCodeProperties.hpp

```diff
@@ -3474,11 +3474,11 @@
    /* .opcode               = */ TR::acmpeq,
    /* .name                 = */ "acmpeq",
    /* .properties1          = */ ILProp1::BooleanCompare,
-   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::SupportedForPRE,
+   /* .properties2          = */ ILProp2::ValueNumberShare | ILProp2::UnsignedCompare | ILProp2::SupportedForPRE,
    /* .properties3          = */ ILProp3::CompareTrueIfEqual,
    /* .properties4          = */ 0,
    /* .dataType             = */ TR::Int32,
-   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer,
+   /* .typeProperties       = */ ILTypeProp::Size_4 | ILTypeProp::Integer | ILTypeProp::Unsigned,
```

Comment:
I think `typeProperties` here describes the type of the result rather than the type of the operands. The way you have it is consistent with the `[il]ucmpxx` opcodes, but I don't know why those include `Unsigned`